packages/mobile/src/identity/saga.ts

```diff
@@ -129,6 +130,10 @@ function* fetchKnownAddresses() {
   }
 }
 
+export function isCeloRewardSender(address: string) {
+  return store.getState().identity.addressToDisplayName[address]?.isCeloRewardSender
+}
+
```

Comment:
This bypasses usual react <-> redux communication and can lead to component not being updated on state change.
We should replace this with a selector.